sdk/Lusid.Sdk.Tests/Features/FeatureExtractor.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Lusid.Sdk.Tests.Features
+{
+
+    public static class FeatureExtractor
+    {
+        public static List<string> GetAllMethodAttributesInNamespace(string nameSpace)
+        {
+            var featureList = Assembly
+                .GetExecutingAssembly()
+                .GetTypes()
+                .Where(t => t.Namespace != null && t.Namespace.ToLower().StartsWith(nameSpace.ToLower()))
+                .SelectMany(t => t.GetMethods()
+                    .Where(m => m.GetCustomAttributes(typeof(LusidFeature), true)?.Length > 0)
+                    .Select(m => m.GetCustomAttributes(typeof(LusidFeature)).Cast<LusidFeature>().First().Code)
+                ).ToList();
+            
+            
+            ValidateFeatures(featureList);
```

Comment:
you can replace this with

```
            if (featureList.GroupBy(x => x).Count(g => g.Count() > 1) > 0)
            {
                throw new DuplicateFeatureException("LusidFeature annotations with duplicate values have been found. " +
                                                    "Please make sure no LusidFeature annotations duplicates are present.");
                
            }
```

Reply:
Decided to print the duplicate values and display them to the user, so they're easier to find

```
            var duplicatesEnumerable = features
                .GroupBy(g => g)
                .Where(x => x.Count() > 1)
                .Select(k => k.Key);
            var duplicateValues = string.Join(", ", duplicatesEnumerable.ToList());
            if(duplicateValues.Length > 0){
                throw new DuplicateFeatureException($"LusidFeature annotations with duplicate values {duplicateValues} have been found. " +
                                                    "Please make sure no LusidFeature annotations duplicates are present.");
            }
```